koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -263,10 +243,18 @@ def get_additional_tags(self):
     def get_missing_label_ids(self):
         data = self.get_data_labels()
         missing_labels = []
-        for label_id in LABELS:
+        # check required labels, have to be defined explicitly and not via env
+        for label_id in LABELS_EXPLICIT_REQUIRED:
             assert label_id in data
             if not data[label_id]:
                 missing_labels.append(label_id)
+
+        # check for all labels, unless required or default values provided,
+        # they should be at least defined (even via env)
+        for label_id in LABELS_ENV_REQUIRED:
+            assert label_id in data
```

Comment:
It would be nicer to raise an exception here (in an `if` block) instead of calling assert (when doing so, we could also remove the other assertion above).

Reply:
this is just really dummy assert because, in get_data_labels all LABELS which includes the ones from LABELS_EXPLICIT_REQUIRED, LABELS_ENV_REQUIRED will be defined

so I just can leave this here, or just remove that assert, I just wanted to keep it consisten